File: C2206L_C2204L/DMA/testapp/testapp/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace testapp.Models
{
    public class Product
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public Double Price { get; set; }
        public int Count { get; set; }
    }
}

File: C2108G2/IASF/de02/src/main/java/com/aptech/de02/de02/models/Product.java
package com.aptech.de02.de02.models;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "products")
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
@Setter
@Data//toString
public class Product {
    @Id
    @Column(name = "product_id")
    private String productID;

    @ManyToOne
    @JoinColumn(name = "category_id")
    private Category category;

    @Column(name = "product_name")
    private String productName;

    @Column(name = "price")
    private int price;

    @Column(name = "description")
    private String description;
}
